tests/test_jsonapi/test_fetching_data.py

```diff
@@ -377,6 +377,25 @@ def test_relationship_links(self):
         assert links['self'].endswith('/article/1/relationships/author')
         assert links['related'].endswith('/article/1/author')
 
+    def test_explicitly_set_primary_key_in_links(self):
```

Comment:
This test is not explicitly testing a feature required by the JSON API specification, so it should live in maybe `tests/test_manager.py`.